src/panels/AssetPanel.php

```diff
@@ -50,7 +49,7 @@ public function getDetail()
      */
     public function save()
     {
-        $bundles = Yii::$app->view->assetManager->bundles;
+        $bundles = Yii::$app->assetManager->bundles;
```

Comment:
While by default these would be equal it can be that a view uses different asset manager.